support-workers/src/main/scala/com/gu/support/workers/lambdas/CreateZuoraSubscription.scala

```diff
@@ -154,106 +152,6 @@ object CreateZuoraSubscription {
       state.acquisitionData
     )
 
-  private def buildSubscribeItem(state: CreateZuoraSubscriptionState, subscriptionData: SubscriptionData): SubscribeItem = {
```

Comment:
all these non-product-specific builder functions are moved to a new file SubscriptionData.scala